Lesson-7/footer.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="heading_logo">
+        <img src="image/smoll_logo.png" alt="heading_logo">
+    </div>
+    <div class="container">
+    <link rel="stylesheet" href="footer.css">
+    <div class="container_about">
+        <ul>
+        <li class="heading_about">About</li>
+        <li class="list">Now Triphouse works</li>
+        <li class="list">Careers</li>
+        <li class="list">Privacy</li>
+        <li class="list">Terms</li>
+    </ul>
+    </div>
+    <div class="container_property_types">
+        <ul>
+        <li class="heading_property">Property types</li>
+        <li class="list">Guest houses</li>
+        <li class="list">Hotels</li>
+        <li class="list">Apartments</li>
+        <li class="list">Villas</li>
+        <li class="list">Holiday homes</li>
+        <li class="list">Hostels</li>
+    </ul>
+    </div>
+    <div class="container_support">
+        <ul>
+        <li class="heading_support">Support</li>
+        <li class="list">Contact Customer Service</li>
+        <li class="list">FAQ</li>
+    </ul>
+    </div>
+    </div>
```

Comment:
Don't hide full list for medium displays
![image](https://user-images.githubusercontent.com/20189313/129097232-cf411f8b-2ef6-4049-b979-67efc5aecee3.png)